ironfish/src/consensus/verifier.ts

```diff
@@ -167,10 +167,8 @@ export class Verifier {
     tx?: IDatabaseTransaction,
   ): Promise<VerificationResult> {
     return this.chain.db.withTransaction(tx, async (tx) => {
-      const noteSize = await this.chain.notes.size(tx)
```

Comment:
Ah, looks like this might have been a bug, good spot! If possible it might be nice to pull this out into a separate PR so any issues with it aren't conflated with the LeavesIndex cache.

Reply:
Moved that fix here, https://github.com/iron-fish/ironfish/pull/798